packages/ui/src/bounty/modals/SubmitWorkModal/SubmitWorkModal.tsx

```diff
@@ -116,63 +116,65 @@ export const SubmitWorkModal = () => {
   return (
     <Modal onClose={hideModal} modalSize="l" modalHeight="xl">
       <ModalHeader title={t('modals.submitWork.title')} onClick={hideModal} />
-      <ModalBody>
-        {state.matches(SubmitWorkStates.generalParameters) && (
-          <RowGapBlock gap={24}>
-            <Row>
-              <RowGapBlock gap={8}>
-                <h4>{t('modals.submitWork.subtitle')}</h4>
-              </RowGapBlock>
-            </Row>
-            <Container
-              disabled
-              label="Bounty ID"
-              tooltipText={t('modals.submitWork.submitWorkInput.bountyId')}
-              inputSize="l"
-            >
-              <TextBig value bold>
-                {modalData.bounty.id}
-              </TextBig>
-            </Container>
-            <Row>
-              <RowGapBlock gap={20}>
-                <SelectedMember
-                  disabled
-                  member={activeMember}
-                  label={t('modals.submitWork.submitWorkInput.memberId')}
-                />
-                <InputComponent
-                  id="field-work-title"
-                  required
-                  inputSize="m"
-                  label={t('modals.submitWork.submitWorkInput.workTitle')}
-                  message={workTitleValidation}
-                  validation={state.context.workTitle.length > 70 ? 'invalid' : undefined}
-                >
-                  <InputText
-                    id="field-work-title"
-                    value={state.context.workTitle}
-                    onChange={(e) => send('SET_WORK_TITLE', { workTitle: e.target.value })}
-                    placeholder={t('modals.submitWork.submitWorkInput.workTitlePlaceholder')}
+      <ScrolledModalBody>
```

Comment:
I think `ScrolledModalBody` is a replacement for `ModalBody` instead of wrapper

Reply:
Yes, plus there's also a container that should be inside `ScrolledModalBody` called `ScrolledModalContainer`. You can have a look at: https://github.com/Joystream/pioneer/pull/2101/commits/abee8f41631eb31d163b22187cad639a3585d69c